rpms/awips2.upc/Installer.ldm/patch/etc/pqact.goesr

```diff
@@ -74,7 +74,7 @@ NIMAGE	^/data/ldm/pub/native/satellite/GOES/([^/]*)/Products/CloudSnow/([^/]*)/(
 # NPP/VIIRS
 #
 # Polar products
-NOTHER	^(TI[A-W]...) (KNES) (......)
```

Comment:
This change is related to the VIIRS change and CIMSS color product, right?

Reply:
yes, to remove the SBN GOES Single Channel data